### ISSUE TITLE
Revisit and document git tag requirement 

### ISSUE THREAD

**Issue by softprops** [softprops/action-gh-release#21]
This can be admittedly confusing to users that aren't aware of GitHib releases constraints.

Let's setup some documentation and align with the way Travis deals with tag constraints for cases where users wish to tag on commit pushes https://docs.travis-ci.com/user/deployment/releases/#setting-the-tag-at-deployment-time

**Comment by joeflack4** [comment]
I agree, and I am confused. I just want to create a new release not on push, but under when the user goes to "Actions" in GitHub and runs the workflow manually. Additionally, after getting this to work, I'm also going to try to set it up so that the action runs weekly.
